### ISSUE TITLE
3.7.0 release: missed dependencies.

### ISSUE THREAD

**Issue by ghost** [tangem/tangem-sdk-android-old#225]
### Version
3.7.0

### Issue:
Can't find dependecy for the 'android' path.

```
dependencies {
    implementation "com.github.tangem.tangem-sdk-android:core: 3.7.0" // Ok
    implementation "com.github.tangem.tangem-sdk-android:android: 3.7.0" // Don't exist
}
```

**Comment by WietseWind** [comment]
Same here at XRPL Labs
@lazutkin-andrey

**Comment by N3TC4T** [comment]
@gbixahue @MamaLemon @lazutkin-andrey 
Can we get this fixed ?

**Comment by kozarezvlad** [comment]
fixed for 3.7.1 version

**Comment by N3TC4T** [comment]
Thanks @kozarezvlad.
